tests/e2e/specs/shopper/cart-checkout/translations.test.js

```diff
@@ -15,29 +15,13 @@ describe( 'Shopper → Cart & Checkout → Translations', () => {
 	// `woocommerce-gutenberg-products-block`. If we try to install the language files
 	// directly, it won't find them because of the slug mismatch.
 	beforeAll( async () => {
-		const resultInstallBlocks = await cli(
-			'npm run wp-env run tests-cli "wp plugin install woo-gutenberg-products-block"'
-		);
-		const resultInstallLanguages = await cli(
-			'npm run wp-env run tests-cli "wp language plugin install woo-gutenberg-products-block fr_FR"'
-		);
-		expect( resultInstallBlocks.code ).toEqual( 0 );
-		expect( resultInstallLanguages.code ).toEqual( 0 );
-		await merchant.changeLanguage( 'fr_FR' );
+		await merchant.changeLanguage( 'nl_NL' );
 	} );
 
 	// We need to clean up here by changing the language back to English
 	// and uninstalling the w.org version of Woo Blocks plugin and the language files
 	afterAll( async () => {
 		await merchant.changeLanguage( 'en_EN' );
```

Comment:
`en_EN` option doesn't exist in the language selector. The value for the default language is empty. This is working but I think we should use the correct value to prevent any confusion.